py_gtfs_rt_ingestion/ingest.py

```diff
@@ -1,78 +1,86 @@
+from py_gtfs_rt_ingestion import Configuration
 
+DESCRIPTION = "Convert a json file into a parquet file. Used for testing."
 
 def parseArgs(args) -> dict:
     parser = argparse.ArgumentParser(description=DESCRIPTION)
     parser.add_argument(
-        '--fname',
-        dest='fname',
+        '--input',
+        dest='input_file',
         type=str,
         required=True,
         help='provide filename to ingest')
 
+    parser.add_argument(
+        '--output',
+        dest='output_dir',
+        type=str,
+        required=True,
+        help='provide a directory to output')
+
     parsed_args = parser.parse_args(args)
 
     return parsed_args
 
 
-def run(filename) -> None:
-    if 'https_cdn.mbta.com_realtime_VehiclePositions_enhanced' not in filename:
-        raise Exception('Requires VehiclePostitions file.')
+def convert_json_to_parquet(input_filename: str, output_dir: str) -> None:
+    """
+    convert in input *.json.gz file into a parquet file
 
-    fname = Path(filename)
-    schema = get_vehicle_schema()
+    * get the configuration information to transform
+    * create a table object
+    * parse datetime from the header of the json file
+    * append to the table for each element the entities list in the json file
+    * write the table
+    """
+    config = Configuration.from_filename(input_filename)
 
-    pa_table_dict = {k.name:[] for k in schema}
+    schema = config.get_schema()
+    table = {key.name:[] for key in schema}
 
-    with gzip.open(fname, 'rb') as f:
+    with gzip.open(Path(input_filename), 'rb') as f:
         json_data = json.loads(f.read())
-        feed_timestamp = json_data['header']['timestamp']
-        feed_dt = get_datetime_from_header(json_data)
-
-        for ent in json_data['entity']:
-            v = ent['vehicle']
-            r = {
-                'year': feed_dt.year,
-                'month': feed_dt.month,
-                'day': feed_dt.day,
-                'hour': feed_dt.hour,
-                'feed_timestamp': feed_timestamp,
-                'vehicle_timestamp': v.get('timestamp'),
-                'vehicle_id': ent.get('id'),
-                'vehicle_label': v["vehicle"].get('label'),
-                'current_status': v.get('current_status'),
-                'current_stop_sequence': v.get("current_stop_sequence"),
-                'stop_id': v.get("stop_id"),
-                'position': v.get('position'),
-                'trip': v.get("trip"),
-                'consist_labels': [],
-            }
-            if v['vehicle'].get('consist') is not None:
-                r['consist_labels'] = [c["label"] for c in v["vehicle"].get("consist")]
-
-            for k in r:
-                pa_table_dict[k].append(r[k])
-
-    pa_table = pa.table(pa_table_dict, schema=schema)
+
+        # parse timestamp info out of the header
+        header = json_data['header']
+        feed_timestamp = header['timestamp']
+        timestamp = datetime.utcfromtimestamp(feed_timestamp)
```

Comment:
i decided to just parse this directly for now. if we need to get more info out of there, we can do that later.